frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.tsx

```diff
@@ -135,21 +168,35 @@ const ExplainerTrigger = () => {
             positionProps={positionProps}
             ref={overlayRef}
           >
-            <p>{l10n.getString("tooltip-email-domain-explanation-part-one")}</p>
-            <br />
-            <p>{l10n.getString("tooltip-email-domain-explanation-part-two")}</p>
-            <br />
-            <p>
-              <Localized
-                id="tooltip-email-domain-explanation-part-three"
-                vars={{ mozmail: "mozmail.com" }}
-                elems={{
-                  p: <p />,
-                }}
-              >
-                <span />
-              </Localized>
-            </p>
+            {hasPremium ? (
+              <>
+                <p>
+                  {l10n.getString("tooltip-email-domain-explanation-part-one")}
+                </p>
+                <br />
```

Comment:
nit: not sure `<br/>` is semantically correct here (but it is prior art and used elsewhere, so feel free to ignore). In fact, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br has a "Note:" for this exact case:

> _**Note:** Do not use `<br>` to create margins between paragraphs; wrap them in [`<p>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p) elements and use the [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS) [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) property to control their size._

Reply:
Leaving as is. Would be better to come back to this as an improvement as it's used in quite a few places. We can call it a clean up!